lib/ansible/plugins/lookup/together.py

```diff
@@ -22,7 +22,7 @@
 
 EXAMPLES = """
 - name: item.0 returns from the 'a' list, item.1 returns from the '1' list
-  debug:
+  ansible.builtin.debug:
     msg: "{{ item.0 }} and {{ item.1 }}"
   with_together:
```

Comment:
Do we want FQCNs here as well?
```suggestion
  with_ansible.builtin.together:
```
(Also present in multiple other lookup examples.)

Reply:
Unsure, I didn't see it as such in the [looping documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html), so I didn't add them in here. If you believe they should be there, I can do another pass through and add them in.

Also, just to clarify, would it be
`ansible.builtin.with_together`
or as you have it
`with_ansible.builtin.together`

Reply:
I'm also not sure, I mainly wanted to bring it up to attention of everyone. It's probably best to not do it in this PR, since this will probably a lot more controversial than all other changes.

In any case, it would be `with_ansible.builtin.together`. The general syntax is `with_<name of lookup>`, and the lookup's name here is `ansible.builtin.together`.

Reply:
Thanks for the clarification. I find myself also agreeing that it might not be best for this PR. The community vote seemed to be scoped to modules, and even though `together` (and others) are technically modules, the presence of the `with` makes me want to treat them as "looping constructs"

That last statement does come down to perspective though. If others feel differently, I can incorporate those changes in as necessary

Reply:
I agree, let's keep this PR as is for now and maybe one of you can open an issue to say we should either use FQCN with the lookup or at least add this to the docs to explain the syntax mentioned here?